ewkcoffea/modules/objects_wwz.py

```diff
@@ -17,32 +17,50 @@ def get_cleaned_collection(obj_collection_a,obj_collection_b,drcut=0.4):
 ######### WWZ 4l analysis object selection #########
 
 # WWZ preselection for electrons
-def is_presel_wwz_ele(ele,tight):
+def is_presel_wwz_ele(ele,year,tight):
     mask = (
         (ele.pt               >  get_ec_param("wwz_pres_e_pt")) &
         (abs(ele.eta)         <  get_ec_param("wwz_pres_e_eta")) &
         (abs(ele.dxy)         <  get_ec_param("wwz_pres_e_dxy")) &
         (abs(ele.dz)          <  get_ec_param("wwz_pres_e_dz")) &
+        (ele.miniPFRelIso_all < get_ec_param("wwz_pres_e_miniPFRelIso_all")) &
         (abs(ele.sip3d)       <  get_ec_param("wwz_pres_e_sip3d")) &
-        (ele.miniPFRelIso_all <  get_ec_param("wwz_pres_e_miniPFRelIso_all")) &
         (ele.lostHits         <= get_ec_param("wwz_pres_e_lostHits"))
     )
-    if tight: mask = (mask & ele.convVeto & (ele.tightCharge == get_ec_param("wwz_pres_e_tightCharge")))
-    return mask
+    mask_2022 = (
+        (ele.pt               >  10.0) &
+        (abs(ele.eta)         <  2.5) &
+        (abs(ele.dxy)         <  0.05) &
+        (abs(ele.dz)          <  0.1)
+    )
+    if ("2022" in year): mask_return = (mask_2022 & ele.mvaIso_WP80)
+    if ("2022" not in year): mask_return = mask
```

Comment:
I think this would be more readable like:
```
if 2022 in year:
    do stuff i.e. set the return mask 
else:
   do other stuff
```
And then you could also put the if tight into the 2022 block (since that only applies to 2022?)